associacao-brasileira-de-normas-tecnicas-ufrgs-note-initials-with-ibid.csl

```diff
@@ -280,542 +496,853 @@
+                  <text variable="page" prefix="p. " suffix="."/>
+                  <text macro="access"/>
+                </group>
+              </else-if>
+              <else-if type="manuscript">
+                <!-- Manuscrito e Partitura -->
+                <group>
+                  <text macro="author" suffix=". "/>
+                  <text macro="title" suffix=" "/>
+                  <choose>
+                    <if variable="medium" match="any">
+                      <group>
+                        <text variable="genre" suffix=". "/>
+                        <text macro="publisher"/>
+                        <text variable="medium" suffix="."/>
+                      </group>
+                    </if>
+                    <else>
+                      <group>
+                        <text variable="publisher-place" suffix=", "/>
+                        <text variable="publisher" suffix=", "/>
+                        <text macro="issued"/>
+                        <text variable="genre" prefix=" " suffix="."/>
+                      </group>
+                    </else>
+                  </choose>
+                  <text macro="access"/>
+                </group>
+              </else-if>
+              <else-if type="map">
+                <!-- Mapa -->
+                <group>
+                  <text macro="author" suffix=". "/>
+                  <text macro="title"/>
+                  <text macro="publisher"/>
+                  <text variable="genre" suffix=". "/>
+                  <text variable="scale" prefix="Escala " suffix=". "/>
+                  <text macro="access"/>
+                </group>
+              </else-if>
+              <else-if type="paper-conference">
+                <!-- trabalho em evento -->
+                <text macro="author" suffix=". "/>
+                <text macro="title" suffix=". "/>
+                <text macro="event"/>
+                <text macro="container-contributors" text-case="uppercase"/>
+                <text macro="secondary-contributors"/>
+                <text macro="container-title"/>
+                <text macro="locators"/>
+                <choose>
+                  <if variable="issued" match="none">
+                    <text macro="publisher"/>
+                    <text variable="page" prefix=" p. " suffix="."/>
+                  </if>
+                  <else>
+                    <text macro="publisher"/>
+                    <text variable="page" prefix=" p. " suffix=". "/>
+                    <text variable="status" suffix=". " font-style="italic" text-case="capitalize-first"/>
+                  </else>
+                </choose>
+                <text macro="access"/>
+              </else-if>
+              <else-if type="patent">
+                <!-- Patente -->
+                <group>
+                  <text macro="author" suffix=". "/>
+                  <text macro="title" prefix=" " suffix=". "/>
```

Comment:
No need for prefix. Also pretty sure ". " is already in the title macro for patent, so this would give you ". . "